packages/api-explorer/src/components/Response/ResponseBody.jsx

```diff
@@ -100,14 +107,14 @@ Unauthorized.defaultProps = {
 function ResponseBody({ result, isOauth, oauth }) {
   return (
     <div style={{ display: 'block'}}>
-      {result.status !== 401 && <Authorized result={result} />}
-      {result.status === 401 && <Unauthorized isOauth={isOauth} oauth={oauth} />}
+      {result.status !== 401 && <Result result={result} />}
```

Comment:
Simplify the whole component by checking for `401` and print the message. Always print the `div` for Result.

This way there's no need for an `Unauthorized` and `Result` component.